opentype/opentype.go

```diff
@@ -0,0 +1,68 @@
+// Package opentype provides an implementation of
+// font.Face for Opentype (Truetype) fonts.
+//
+// It supports advanced Opentype and AAT layout features.
+package opentype
+
+import "github.com/go-text/font"
+
+// var _ FaceOpentype = (*Face)(nil)
+
+// FaceOpentype is a specialization of font.Face for Opentype
+// font files.
+type FaceOpentype interface {
+	font.Face
+
+	// TablesLayout fetches the advanced Opentype and AAT layout tables of the font.
+	TablesLayout() TablesLayout
+
+	// Variations returns the variations for the font,
+	// or an empty table.
+	Variations() TableFvar
+
+	// IsGraphite returns true if the font has Graphite capabilities.
+	// The returned Face will be used to load Graphite tables.
+	// Overide this method to disable Graphite functionalities.
+	IsGraphite() (bool, *Face)
```

Comment:
```suggestion
	IsGraphite() (*Face, bool)
```
(would better match the common comma-ok idiom)